packages/berlin/src/pages/Holding.tsx

```diff
@@ -0,0 +1,47 @@
+// React and third-party libraries
+import { useQuery } from '@tanstack/react-query';
+
+// Components
+import { FlexColumn } from '../components/containers/FlexColum.styled';
+import { Body } from '../components/typography/Body.styled';
+import Link from '../components/link';
+import { useNavigate } from 'react-router-dom'; // Import useNavigate
+
+// API Calls
+import { fetchEvents } from 'api';
+
+function Holding() {
+  const navigate = useNavigate();
+
+  const { data: events, isLoading: eventsLoading } = useQuery({
+    queryKey: ['events'],
+    queryFn: fetchEvents,
+  });
+
+  if (eventsLoading) {
+    return <Body>Loading events...</Body>;
+  }
+
+  // Define onClick handler to navigate to registration page and trigger reload
+  const handleRegistrationClick = () => {
+    navigate(`/events/${events?.[0]?.id}/register`, { replace: true }); 
+    window.location.reload();
```

Comment:
In theory, react query handles this when we invalidate the queries, but as you can see in the posted video this is not working? My registration fields didn't updated. doing a window.location.reload() is a bit agressive as it "messes" with our single page application user experience where everything is smooth, lets check what happened here.

cc @diegoalzate

Reply:
I agree, a hard reload might cause an unexpected state. If it does not work without the reload then we should think about a different approach

Reply:
Issue: https://github.com/lexicongovernance/forum-frontend/issues/146